Sources/MongoMobile/MongoMobile.swift

```diff
@@ -76,16 +85,20 @@ public class MongoMobile {
      * Perform required operations to clean up the embedded server.
      */
     public static func close() throws {
+        self.embeddedClients.forEach { ref in
+            ref.reference?.close()
+        }
+
         let status = mongo_embedded_v1_status_create()
         for (_, instance) in embeddedInstances {
             let result = mongo_embedded_v1_instance_destroy(instance, status)
-            if result == MONGO_EMBEDDED_V1_ERROR_EXCEPTION.rawValue {
```

Comment:
were these values not actually available? If so, I would prefer for us to redefine them in the code itself. They have useful, descriptive names

Reply:
they are available, but I guess we were only checking for a single value before, whereas anything non-zero is an error. 
maybe we should come up with some way of consistently mapping these codes to different `MongoMobileError` cases

Reply:
I can include that in this PR.

Previously, we were ignoring all other errors, which caused issues in that we had false positive cleanups.